opsdroid/core.py

```diff
@@ -112,7 +114,13 @@ def critical(self, error, code):
     @staticmethod
     def handle_async_exception(loop, context):
         """Handle exceptions from async coroutines."""
-        _LOGGER.error(_("Caught exception"))
+        if "future" in context:
```

Comment:
This is interesting and seems unrelated to this PR. Happy for it to be here but you could comment a little on what it is and why it's here?

Reply:
So this I put here because I wasn't getting the traceback for an error and this is the way I worked out how to make it log the full traceback.